tutorials/beginner/configuration.md

```diff
@@ -0,0 +1,122 @@
+Scroll down or use the "Search" field to find the "Network Binding". Click on install, the binding will be downloaded and installed automatically.
+
+![](images/picture_06.jpg)
+
+![](images/picture_07.jpg)
+
+Now that the binding is installed, switch back to the "Inbox". If you click the "+"-button, you now are able to choose the binding which you want to use for discovering new things. Because we only installed the network binding so far, it's the only binding to choose.
+
+![](images/picture_08.jpg)
+
+As soon as you click on "Network Binding", the binding will start an automatic discovery of all (reachable) network devices in the same subnet as your openHAB 2 installation resides in via ICMP (ping).
+
+![](images/picture_09.jpg)
+
+The discovered "things" are now displayed in the inbox, and you can decide what to do next:
+
+* add the device as a thing by clicking the check icon
+* ignore the device by clicking the crossed-out-eye icon
+* delete the device by clicking the trashcan icon
+
+The difference between ignoring and deleting discovered things is as follows:
+
+* if you ignore a thing, it disappears from the visible inbox. If you decide to add it at a later time, you can do so by scrolling to the bottom of the inbox and click the "SHOW IGNORED" button.
+* if you delete the thing, it won't be visible in the inbox any more. But if you start a new discovery process later, the thing will reappear.
+
+In this example, we click on the check icon of the thing with the IP address 192.168.1.103. Let's assume this is John's mobile phone which we want to use for presence detection. We change the (automatically added during the discovery) name of the thing on the next page to "Presence_Mobile_John" and click on the "ADD AS THING" button.
+_Please note: if your network devices use a DHCP server to obtain ther IP addresses automatically, please make sure to define a DHCP reservation for a device which you want to use in openHAB 2. Otherwise it's IP address may change in the future and your thing definiton won't work any more.  Information about how to define a DHCP reservation can be normally be found in the manual of your network router._
+
+![](images/picture_10.jpg)
+
+_Note: Thing names have to be unique! You should follow a certain naming convention concerning your things. In this example, we used the purpose of the thing, "Presence", followed by the device type, "_Mobile", and finally the owner's name, "_John", to contruct the final thing name. Another suggestion for different things, for example a wallplug which is plugged in in the living room in first floor providing power to a TV, is to use the type, the floor, the room and the usage of the thing. In this case it would be "Wallplug_FF_LR_TV"._ 
```

Comment:
I am not talking about things. Thing labels as well as item labels should hold a meaningful name without underscore and such.

I agree with your fist comment but wanted to justify the idea behind the note and pointed out that this whole paragraph can be moved (instead of being deleted) over to the part where items are created manually in files.